Packs/SumoLogic_Cloud_SIEM/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,26 @@
+#### Integrations
+##### Sumo Logic Cloud SIEM
+Added the followings:
+
+#### Incident Types
+##### Sumo Logic Insight 
+- New incident type corresponding to Sumo Logic Cloud SIEM Insight
+
+#### Incident Fields
+##### Sumo Logic Cloud SIEM Insight Signals
+- **Sumo Logic Insight Signals** field corresponding to all Sumo Logic Cloud SIEM Signals contributing to the Insight
+##### Sumo Logic Cloud SIEM Insight Entity
+- **Sumo Logic Cloud SIEM Insight Entity** field corresponding to the Sumo Logic Cloud SIEM Entity associated with the Insight 
+##### Sumo Logic Cloud SIEM Insight Involved Entities
+- **Sumo Logic Cloud SIEM Insight Involved Entities** field corresponding to all entities involved in the Insight
+##### Sumo Logic Cloud SIEM Insight URL 
+- **Sumo Logic Cloud SIEM Insight URL** field contains the URL of the Insight on the Cloud SIEM side
+
+#### Layouts
+##### Sumo Logic Insight
+- New layout for Sumo Logic Insights (Available from Cortex XSOAR 6.2.0).
+
+#### Mappers
+##### Sumo Logic Insight - Incoming Mapper
+- New incoming mapper for Sumo Logic Insight (Available from Cortex XSOAR 6.2.0).
+
```

Comment:
No release note entry was found for the mapper "Sumo Logic Insight - Mapper" in the SumoLogic_Cloud_SIEM pack. This happens cause the RN contains "Incoming Mapper" instead. @idovandijk What do you think? Should we change to Mapper?

No release note entry was found for the classifier "Sumo Logic Insight - Classifier" and
 Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Reply:
I think the RN should generate what the name of the mapper is. They could choose to change the name of the mapper to Incoming Mapper, but then I believe the SDK would change it to `Sumo Logic Insight - Incoming Mapper - Incoming Mapper`, wouldn't it?

Reply:
@idovandijk : the SDK uses "Sumo Logic Insight - Mapper", should I keep it?

Reply:
@JasBeilin I kept having issues with the update-release-notes command until today. It seems paths are particularly  important for some demisto-sdk commands. Having a specific example in online doc page would be great (e.g https://xsoar.pan.dev/docs/documentation/release-notes)

Reply:
Updated the RN